js/audiOrbits.js

```diff
@@ -458,7 +705,7 @@ var audiOrbits = {
 		print("texture loaded.")
 
 		// create camera
-		self.camera = new THREE.PerspectiveCamera(sett.field_of_view, window.innerWidth / window.innerHeight, 1, 3 * sett.scaling_factor);
+		self.camera = new THREE.PerspectiveCamera(sett.field_of_view, window.innerWidth / window.innerHeight, 1, 3.5 * sett.scaling_factor);
```

Comment:
Oops I forgot I had made this change. I was experimenting with it to try to prevent orbital levels from just "appearing" in the background. This should be reverted.